osu.Game.Tests/Visual/Gameplay/TestSceneSongProgress.cs

```diff
@@ -45,10 +45,7 @@ private void load()
                 },
                 gameplayClockContainer = new MasterGameplayClockContainer(Beatmap.Value, skip_target_time)
                 {
-                    Child = frameStabilityContainer = new FrameStabilityContainer
-                    {
-                        MaxCatchUpFrames = 1
-                    }
+                    Child = frameStabilityContainer = new FrameStabilityContainer()
```

Comment:
We may want to look at exactly why this was limiting to one frame just to be certain though.

Reply:
https://github.com/ppy/osu/commit/5ead85f461eb99775896d09f3931005e0857d030

This was just done for visual testing purposes (to actually show the game catching up in the test scene). I've added back a `Thread.Sleep` to make this still look correct.